docs/usage/configuration-options.md

```diff
@@ -1999,6 +1999,10 @@ For example, if you want to group packages which are older than 2 months:
 The `matchCurrentAge` string must start with one of `>`, `>=`, `<` or `<=`.
 Only one date part is supported, e.g. you _cannot_ do `> 1 year 1 month` and instead should do `> 13 months`.
 
+<!-- prettier-ignore -->
+!!! note
+    Only the words hour(s)/hrs/h, day(s)/d, week(s), month(s)/M and year(s)/y are supported.
```

Comment:
I think there's a typo somewhere. `Months` is `M`, but `Years` is `y`?

I would expect one of these:

1. Months is `M`, years is `Y`
2. Months is `m`, years is `y`
3. Or even: `M/Y` or `m/y`, casing does not matter

Reply:
Let's use the full words in our docs and point the reader to the library's documentation if they want to know all the words.

Reply:
I left the link to the docs out because months is out addition and not supported by `ms`.